### ISSUE TITLE
yarn: error: no such option: --production

### ISSUE THREAD

**Issue by anmol26s** [Chocobozzz/PeerTube#1079]
A user on YUNoHost reported a log having an error `yarn: error: no such option: --production`. I can not reproduce the error. Any idea?

https://github.com/YunoHost-Apps/peertube_ynh/issues/40#issue-360656546

**Comment by Chocobozzz** [comment]
Another package provides the yarn command in debian https://packages.debian.org/search?suite=stretch&section=all&arch=any&searchon=contents&keywords=yarn

The user needs to install yarn from the custom repository: https://github.com/Chocobozzz/PeerTube/blob/develop/support/doc/dependencies.md#debian--ubuntu-and-derivatives

**Comment by rigelk** [comment]
@Chocobozzz the yarn version is correct though. And yarn is installed according to our guide: https://github.com/YunoHost-Apps/peertube_ynh/blob/f5101da647d54ba4ff02ae7c8b39721dd52c3f86/scripts/install#L98-L100

**Comment by anmol26s** [comment]
@rigelk is right yarn version is  latest and installed from `deb https://dl.yarnpkg.com/debian/ stable main`.

**Comment by alexandre-k** [comment]
Hey guys,
I got the same problem, and apparently it is due to the cmdtest package being installed and giving the wrong yarn.
https://stackoverflow.com/a/60016407. 
And install node 12 instead of the latest version (currently 16.x):
https://computingforgeeks.com/how-to-install-nodejs-on-ubuntu-debian-linux-mint/

TLDR: Quick fix:
```
sudo apt remove cmdtest yarn nodejs
curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
sudo apt -y install curl dirmngr apt-transport-https lsb-release ca-certificates
curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
sudo apt update
sudo apt install -y yarn nodejs
```

**Comment by LuckyLukeCH** [comment]
Works perfect, thank you!
